src/App.js

```diff
@@ -4,19 +4,31 @@ import data from './data/dummy.json';
 
 const App = () => {
   return (
-    <div>
-      <ol>
-        {data.map((question) => {
-          return <div>
-            <li key={question.id}>{question.text}</li>
-            <ul>{question.answers.map((answer) => {
-              return <li key={answer.id}>{answer.text}</li>
+    <form>
+      {data.map(question => {
+        return (
+          <fieldset>
+            <legend key={question.id}>{question.text}</legend>
```

Comment:
The `fieldset` above is the element that should have a React `key` attribute, not this one